resources/js/dashboard/pages/administrator/components/Menu.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="col-sm-2  side-menu">
+    <div class="header-menu">
+      <div class="d-flex justify-content-center my-3">
+        <img alt="Avatar" class="avatar" src="../../../../../../public/img/admin/avatar_placeholder.png" />
+      </div>
+      <h5 class="h6 text-center">Administrateur</h5>
+    </div>
+    <div class="parent" v-for="item in items" :key="item.key" @mouseover="hover = item.key" @mouseleave="hover = -1">
+      <div class="elmt" :class="{active : activeItem == item.key || hover == item.key}">
+        <b-link class="elmt nav-link" :to="{name:item.path}" @click="switchActivePage(item.key)">{{item.label}}</b-link> 
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    data () {
+      return {
+        activeItem: 1,
+        hover: -1,
+        items: [
+          {key: 1, label: "Utilisateurs", path: 'administrator.users'},
+          {key: 2, label: "Situation Epidemiologique", path: 'administrator.users.create'}
+        ]
+      }
+    },
+    methods: {
+      switchActivePage (key) {
+        this.activeItem = key
+      }
+    }
+  }
+</script>
+
+<style>
```

Comment:
ce style doit être scoped, il a des effets de bord sur tous les autres style de l'application 
```suggestion
<style scoped lang='scss'>
```

Reply:
Et toujours faire du mieux que possible le sass